### PR TITLE
Amqp response error

### PR DESCRIPTION
**Pull request by akolomietc**
@s0rg i don't understand your language )),  but i did it

### CODE REVIEW

**Review by s0rg — COMMENTED** [review]
Добавь еще тест на возврат ошибки из обработчика

File: amqp-kit/errors.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/streadway/amqp"
+)
+
+type Error struct {
```
Comment (avdva) [inline amqp-kit/errors.go:12]: maybe, comment all exported entities?

Comment (akolomietc) [reply amqp-kit/errors.go:12]: yes, fixed!

File: amqp-kit/errors.go [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	b, err := json.Marshal(Response{Error: e})
+	if err != nil {
+		return
+	}
+	pub.Body = b
+
+	ch.Publish(replyExchange, replyTo, false, false, *pub)
+}
+
+func ReplyAndAckErrorWithCodeEncoder(ctx context.Context, err error, d *amqp.Delivery, ch Channel, pub *amqp.Publishing) {
```
Comment (avdva) [inline amqp-kit/errors.go:67]: comment?

Comment (akolomietc) [reply amqp-kit/errors.go:67]: fix

**Review by s0rg — APPROVED** [review]

**Pull request merged**